tests/ArrayProxy/CMakeLists.txt

```diff
@@ -34,4 +34,7 @@ if (NOT TESTS_BUILD_ONLY_DYNAMIC)
 
 	set_target_properties(ArrayProxy PROPERTIES FOLDER "Tests")
 	target_link_libraries(ArrayProxy "${Vulkan_LIBRARIES}")
+	if (UNIX)
```

Comment:
In the samples, handling such warnings in done in the cpp files, instead of the CMakeLists.txt, like so (from 02_EnumerateDevices.cpp):

#if defined( _MSC_VER )
// no need to ignore any warnings with MSVC
#elif defined( __clang__ )
#  pragma clang diagnostic ignored "-Wunused-variable"
#elif defined( __GNUC__ )
#  pragma GCC diagnostic ignored "-Wunused-but-set-variable"
#else
// unknow compiler... just ignore the warnings for yourselves ;)
#endif

Would you mind adjusting your fix accordingly?